hydra/app/models/acda.rb

```diff
@@ -119,12 +119,12 @@ def assign_id
   # Policy Area property
   property :policy_area, predicate: ::RDF::Vocab::DC.subject, multiple: true do |index|
     index.as :stored_searchable, :stored_sortable, :facetable
-  end  
+  end
 
   # DC subject
   # ==============================================================================================================
   # Topic property
-  property :topic, predicate: ::RDF::URI.intern('http://purl.org/dc/terms/subject'), multiple: true do |index|
+  property :topic, predicate: ::RDF::URI.intern('http://lib.wvu.edu/hydra/subject'), multiple: true do |index|
```

Comment:
this is the only actual change in this file, everything else is just removing whitespace